.gitlab-ci.yml

```
@@ -538,6 +538,33 @@ test-suite:base:dev:
     expire_in: 2 week
   allow_failure: true
 
+test-suite:edge+coverage:
+  stage: build
+  script:
+    - ulimit -s 16384
+    - ./configure -prefix $(pwd)/_build/install/coverage -native-compiler no
+    - make dunestrap
+    - dune build --instrument-with bisect_ppx --workspace dev/dune-workspace.bisect @runtest
+    - find _build/coverage/ -name '*.coverage' -size 0 -exec rm {} \; # workaround truncated coverage file bug
+    - bisect-ppx-report html --source-path=_build/coverage --coverage-path=_build/coverage -o _build/coq-coverage --title="Coq's Test suite"
+    - bisect-ppx-report coveralls coverage.json --source-path=_build/coverage --coverage-path=_build/coverage --git --service-name="github" --service-pull-request="${CI_COMMIT_BRANCH#pr-}" --service-job-id="$CI_JOB_ID"
+    - wget -q https://uploader.codecov.io/v0.1.0_4653/linux/codecov
+    - chmod +x codecov
+    - ./codecov -t ${CODECOV_TOKEN} -f coverage.json
+    # Bah bisect-ppx doesn't support gitlab CI
+    # - bisect-ppx-report send-to --source-path=_build/coverage --coverage-path=_build/coverage Codecov
```

Should this be uncommented to test?

---

I think the line just above `./codecov ....` should do the same trick, but I forgot about how the process looks like.

Maybe we should check if the `bisect-ppx-report` has support for gitlab CI already in the newer version.

---

I've made https://github.com/aantron/bisect_ppx/pull/436

---

What about doing
```suggestion
    - bisect-ppx-report send-to --source-path=_build/coverage --coverage-path=_build/coverage Coveralls --git --service-name="github" --service-pull-request="${CI_COMMIT_BRANCH#pr-}" --service-job-id="$CI_JOB_ID"
    - bisect-ppx-report send-to --source-path=_build/coverage --coverage-path=_build/coverage Codecov --git --service-name="github" --service-pull-request="${CI_COMMIT_BRANCH#pr-}" --service-job-id="$CI_JOB_ID"
```
?